apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-card.js

```diff
@@ -57,8 +58,21 @@ function WelcomeTourCard( {
 	return (
 		<Card className="welcome-tour-card" isElevated>
 			<CardOverlayControls onDismiss={ onDismiss } onMinimize={ onMinimize } />
-			<CardMedia>
-				<img alt={ __( 'Editor Welcome Tour', 'full-site-editing' ) } src={ imgSrc } />
+			{ /* TODO: Update selector for images in @wordpress/components/src/card/styles/card-styles.js */ }
+			<CardMedia className="welcome-tour-card__media">
+				<picture>
+					{ imgSrc.mobile && (
+						<source
+							srcSet={ imgSrc.mobile.src }
+							type={ imgSrc.mobile.type }
+							media={ getMediaQueryList( '<480px' ).media }
```

Comment:
Technically speaking `getMediaQueryList` may return `undefined`. I guess we could guard irrespective if we want to be extra cautious. 🤷

Reply:
Thanks, @chriskmnds! I added a guard to be catch errors when non-supported breakpoints are used.